scss/components/_themeforest.scss

```diff
@@ -0,0 +1,85 @@
+.themeforest {
+  background: $color-2;
+
+  &__content {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__item {
+    display: inline-block;
+    max-width: 380px;
+    font-family: PTSans-Bold, AdobeInvisFont, MyriadPro-Regular;
+  }
+
+  &__heading {
+    position: relative;
+    color: #40474e;
+    font-size: 12pt;
+    text-transform: uppercase;
+    margin: 60px 0 65px;
+    font-family: $heading-font-stack;
+    font-size: 15pt;
+
+    &:after {
+      content: "";
+      position: absolute;
+      left: 20px;
+      right: 20px;
+      bottom: -25px;
+      height: 2px;
+      opacity: 0.06;
+      background: linear-gradient(to right, black, #b8b8b8);
+    }
+  }
+
+  &__button {
+    font: normal $medium-weight 18px $basic-font-stack;
+    border: 0 none;
+    color: white;
+    border-radius: 20px;
+    padding: 10px 10px;
+    margin: 10px 0 60px 0;
+  }
+
+  &__image {
+    -webkit-clip-path: inset(123px 211px 2px 22px);
```

Comment:
if you looked into gulpfile you could saw autoprefixer there - so there is no need for `-webkit-clip-path`

Reply:
Agree. Thank you for noticing.